README.md

```diff
@@ -51,12 +53,7 @@ If the "text" scope is selected, the program will identify student errors in the
 If the "image" scope is selected, the program will identify issues in submission images, optionally comparing them to reference solutions. Question numbers can be specified by adding the tag `markus_question_name: <question name>` to the metadata for the code cell that generates the submission image. The previous cell's markdown content will be used as the question's context.
 
 ## Submission Type
-The program automatically detects submission type based on file extensions in the assignment directory:
```

Comment:
This isn't good; you've reverted previous changes that were committed to the master branch. This is likely due to a bad git merge.

The easiest way to resolve this is to first identify exactly which commits include your new changes, and then make a new branch _off of main_, and use [`git cherry-pick`](https://git-scm.com/docs/git-cherry-pick) to copy over your commits from this original branch.